projects/igniteui-angular/src/lib/core/styles/components/input/_input-group-theme.scss

```diff
@@ -1454,8 +1454,10 @@
         padding-inline: pad-inline(rem(2px), rem(4px), rem(6px));
     }
 
-    %form-group-input--search {
-        padding-inline: pad-inline(rem(6px), rem(8px), rem(10px));
+    @if $variant == 'indigo-design' {
+        %form-group-input--search {
+            padding-inline: pad-inline(rem(6px), rem(8px), rem(10px));
+        }
```

Comment:
Is this intentional in the PR or is it a left over from another branch? I see that this PR specifically introduces changes to the badge typography.

Reply:
Yes, it is intentional. It fixes a bug introduced with the Indigo design changes PR (#13948). If needed I will make a separate PR for this.

Reply:
Sneaky, but ok.